src/common/log.ts

```diff
@@ -49,6 +45,14 @@ export class Log {
     }
 
     /**
+     * Logs a warning message to the console.
+     */
+    public static logWarning(message: string, error?: any, outputChannel?: OutputChannel, logStack = true) {
+        // TODO: Refactor this code and create a better implementation
```

Comment:
My intention was actually to leave them on master. @meenakb was working on refactoring the error code, so I think she'll probably provide a fix for this on one of her PRs. If she doesn't, then I'll fix this on a further PR after she merges her changes. Does that sound good to you?